docs/languages.rst

```diff
@@ -8,10 +8,10 @@ written in C.
 
 All language interfaces expose the same core functionality from the
 :doc:`C API <./languages/c-api>`.
-Additional "quality of life" features such as the ability to download
-BridgeStan's source code automatically or compile models from inside the language
-(rather than manually calling :command:`make`) are available on a best-effort basis.
-If you are missing these features in your favorite language, we would welcome a
+
+
+Additional "quality of life" features are available on a best-effort basis.
+If you are missing a feature from one interface in your favorite language, we would welcome a
```

Comment:
> If you are missing a feature from one interface in your favorite language, we would welcome a contribution 

Can we drop the word would?

I'm also hesitant about the word one.  Maybe the?  Other thoughts?

Reply:
Rewrote the whole sentence to "If BridgeStan's bindings to your favorite language are missing one of these features, we welcome contributions!"